app/services/email_checker.rb

```diff
@@ -83,4 +86,8 @@ def mx_records?
       Rails.configuration.mx_checker.records?(domain)
     end
   end
+
+  def sendgrid_api
+    @sendgrid_api ||= SendgridApi.new
```

Comment:
Yes I know, went with this approach initially because I assume we are using Puma with forking, if that is the case we should use the `after_fork` method in the puma config